src/wdio/browser.ts

```diff
@@ -65,4 +65,9 @@ export class Browser {
     public static async waitUntil(condition: () => boolean | Promise<boolean>, timeout: number = DefaultTimeout.SLOW_WAIT): Promise<void> {
         await browser.waitUntil(condition, {timeout});
     }
+
+    //Keyboard functions
```

Comment:
Despite I am OK with the new method added, I would suggest the following to improve/polish the proposed implementation:
- Move this method together with the block of "keyboard control" functions (defined on the top of the class)
- Rename it to `writeText(text: string)`, as write may be too generic